### PR TITLE
MISUV-1061 - Updated submission logic for allowing property to have a…

### PR DESCRIPTION
**Pull request by AlexRimmerHMRC**
… tax year of next tax year

### CODE REVIEW

File: it/connectors/stubs/MultipleIncomeSourcesSubscriptionAPIStub.scala [hunk 1]
```diff
@@ -24,7 +24,7 @@ object MultipleIncomeSourcesSubscriptionAPIStub extends WireMockMethods {
 
   def stubPostSubscription(mtdbsa: String, request: BusinessSubscriptionDetailsModel)
                           (responseCode: Int,
-                           response: JsValue = Json.parse("{}")): StubMapping = {
+                           response: JsValue = Json.obj()): StubMapping = {
```
Comment (scullythornton1) [inline it/connectors/stubs/MultipleIncomeSourcesSubscriptionAPIStub.scala:27]: why does this need to be changed?

File: it/connectors/stubs/IncomeTaxSubscriptionConnectorStub.scala [hunk 2]
```diff
@@ -39,7 +39,7 @@ object IncomeTaxSubscriptionConnectorStub extends WireMockMethods {
       .thenReturn(Status.OK, CacheMap(SessionId, fullSubscriptionDataBothPost))
   }
 
-  def stubFullSubscriptionAllPost(): Unit = stubSubscriptionData(fullSubscriptionDataAllPost)
+  def stubFullSubscriptionGet(): Unit = stubSubscriptionData(fullSubscriptionDataAllPost)
```
Comment (scullythornton1) [inline it/connectors/stubs/IncomeTaxSubscriptionConnectorStub.scala:42]: why does this stub change name to Get?

File: it/connectors/stubs/MultipleIncomeSourcesSubscriptionAPIStub.scala (hunk 1 above)
Comment (AlexRimmerHMRC) [reply it/connectors/stubs/MultipleIncomeSourcesSubscriptionAPIStub.scala:27]: It's simpler and does the same thing.

File: it/connectors/stubs/IncomeTaxSubscriptionConnectorStub.scala (hunk 2 above)
Comment (AlexRimmerHMRC) [reply it/connectors/stubs/IncomeTaxSubscriptionConnectorStub.scala:42]: It was causing confusion during my tests so I renamed it to make sense to what it was doing

**Pull request merged**